pyproject.toml

```diff
@@ -0,0 +1,156 @@
+# https://peps.python.org/pep-0517/
```

Comment:
this file replaces setup.py ... and also conveniently serves as configuration for pretty much everything.  There are links throughout to the relevant info